libraries/engage/product/components/ProductImage/index.jsx

```diff
@@ -177,7 +177,13 @@ class ProductImage extends Component {
 
     // Return the actual image.
     return (
-      <SurroundPortals portalName={PORTAL_PRODUCT_IMAGE}>
+      <SurroundPortals
+        portalName={PORTAL_PRODUCT_IMAGE}
+        portalProps={{
```

Comment:
wouldn't it be more sustainable and useful to pass along all portalProps, in case we need the props in the future to adjust something else with the image?